README.md

```diff
@@ -4,37 +4,61 @@
 
 This repository contains the assets required to build the [Kubernetes website and documentation](https://kubernetes.io/). We're glad that you want to contribute!
 
-## Running the website locally using Hugo
+# Using this repository
 
-See the [official Hugo documentation](https://gohugo.io/getting-started/installing/) for Hugo installation instructions. Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
+You can run the website locally using Hugo, or you can run it in a container runtime. We strongly recommend using the container runtime, as it gives deployment consistency with the live website.
```

Comment:
Getting back to this ...
Since this file has been updated a number of times, it would be good to get these changes made so that the file does not have to be updated (and translated) for some time.

Feedback (minor cleanup): It reads as if the running in a container is not local.
You can run the website locally using Hugo or in a container runtime. We strongly recommend using the container runtime, as it gives deployment consistency with the live website.